src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -108,6 +110,8 @@ InitializeExtensionsResponse handleExtensionInitRequest(InitializeExtensionsRequ
         logger.info("Registering Extension Request received from OpenSearch");
         InitializeExtensionsResponse initializeExtensionsResponse = new InitializeExtensionsResponse(extensionSettings.getExtensionName());
         opensearchNode = extensionInitRequest.getSourceNode();
+        port = extensionInitRequest.getPort();
+        hostAddress = opensearchNode.getHostAddress();
```

Comment:
See similar comment on https://github.com/opensearch-project/OpenSearch/pull/4197 about this being redundant with a field in `opensearchNode`, so it doesn't need a separate field here.

Reply:
Keeping it separate here for better availability to extension developers as [this](https://github.com/opensearch-project/opensearch-sdk/pull/67/files#diff-da305de522ebd01a3b7821d9d3ee90d514a0b92d29a272fd5bb09bcb3a3e2c16R42) method requires `hostAddress` and `port`